src/server/game/Entities/Player/Player.cpp

```diff
@@ -16005,7 +16005,7 @@ void Player::SendQuestUpdate(uint32 questId)
     }
 
     UpdateForQuestWorldObjects();
-    SendUpdatePhasing();
+    UpdateAreaAndZonePhase();
```

Comment:
This was previously a target of discussion in another PR - by forcing the update here you break quests that should not update phase here